check/src/typecheck.rs

```diff
@@ -42,6 +43,8 @@ pub enum TypeError<I> {
     Rename(::rename::RenameError),
     /// Multiple types were declared with the same name in the same expression
     DuplicateTypeDefinition(I),
+    /// A field were defined more than once in a record constructor or pattern match
```

Comment:
`"A field that was defined more than once in a record constructor or pattern match" does not seem quite right?